src/metabase/channel/models/channel.clj

```diff
@@ -37,6 +37,13 @@
   (or (mi/superuser?)
       (perms/current-user-has-application-permissions? :setting)))
 
+(defmethod mi/can-read? :model/Channel
+  [channel]
+  (or (mi/superuser?)
+      (case (:type channel)
+        :channel/http (perms/current-user-has-application-permissions? :setting)
```

Comment:
we only have http channels for now, but I expect other channels will soon be migrated